apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/NotifySupportedNameServers/NotifySupportedNameServers.tsx

```diff
@@ -34,11 +33,11 @@ export const NotifySupportedNameServers = () => {
 
         <Text style={styles.text}>{strings.resolverNoticeText}:</Text>
 
-        <NameServer text={nameServerName[Resolver.NameServer.Handle]} />
+        <NameServer text="ADA Handle" />
 
-        <NameServer text={nameServerName[Resolver.NameServer.Unstoppable]} />
+        <NameServer text="Unstoppable Domains" />
 
-        <NameServer text={nameServerName[Resolver.NameServer.Cns]} />
```

Comment:
New product spec. 

The text is different that the one in the input helper, it's not computable like the helper and its going to be removed at some point so it has been hardcoded.